addons/test_mail/tests/test_performance.py

```diff
@@ -253,7 +253,7 @@ def test_adv_activity_mixin(self):
 
     @users('__system__', 'emp')
     @warmup
-    @mute_logger('odoo.addons.mail.models.mail_mail')
+    @mute_logger('odoo.tests', 'odoo.addons.mail.models.mail_mail')
```

Comment:
why?

Reply:
As seen in first commit, this is to have a cleaner log, similar to what is done for other tests in the field (eg. `test_complex_mail_mail_send`).

This particular case is to avoid `odoo.tests: skip sending email in test mode`

Reply:
Ho, great!